test/browser_testing/features/closing_disclosure.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Test the default values in the Clsoing Disclosure page
+  As a first time visitor to the Owning a Home page
+  I want to have content loaded
+  So that I can get clever and conquer the world
+
+Background:
+   Given I navigate to the "Closing Disclosure" page
+
+@closing_disclosure
+Scenario Outline: Test that tabs are on the page
+   Then I should see "<tab_name>" tab
+
+Examples:
+  | tab_name    |
+  | Checklist   |
+  | Definitions |
+
+
+@closing_disclosure
+Scenario Outline: Test that the content is loaded
+  When I click the tab "<tab_name>"
+  Then Content image is loaded
+
+Examples:
+  | tab_name    |
+  | Checklist   |
+  | Definitions |
+
+
+@closing_disclosure
+Scenario Outline: Test that resizing window size changes image size too
+  When I click the tab "<tab_name>"
+    When I resize window image "<css_selector>" size changes too
+
+Examples:
+  | tab_name    | css_selector          |
+  | Checklist   | div.illustration img  |
+  | Definitions | img.image-map_image   |
+
```

Comment:
Yes, both has to be the same, but I think more specific like div.illustration img.image-map_image